view/frontend/web/template/vue-hawksearch/components/product-price.html

```
@@ -0,0 +1,431 @@
+<span v-if="priceData">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)